google-cloud-storage/acceptance/storage/file_encryption_test.rb

```diff
@@ -159,8 +159,14 @@
   end
 
   describe "KMS customer-managed encryption key (CMEK)" do
-    let(:kms_key) { "projects/helical-zone-771/locations/#{bucket_location}/keyRings/ruby-test/cryptoKeys/ruby-test-key-1" }
-    let(:kms_key_2) { "projects/helical-zone-771/locations/#{bucket_location}/keyRings/ruby-test/cryptoKeys/ruby-test-key-2" }
+    let(:kms_key) {
+      ENV["GCLOUD_RUBY_TEST_KEY_1"] ||
```

Comment:
Alas, the values are currently different. The default keys in #2518 are in a different project, and are set to rotate, so the usage is different. But it might be a good idea to make the environment variable names a bit more explicit, like `GCLOuD_RUBY_STORAGE_TEST_KEY_1`?

Reply:
I would use `GCLOUD_TEST` instead of `GCLOUD_RUBY`, to match the other test-specific environment variables. So perhaps `GCLOUD_TEST_STORAGE_KMS_KEY`?

We might also want to rename the other environment variables to indicate what service those KMS keys are for. So rename `GCLOUD_TEST_KMS_KEY` to `GCLOUD_TEST_BIGQUERY_KMS_KEY`.